bridge/src/main/java/io/stargate/bridge/service/SchemaHandler.java

```diff
@@ -158,6 +158,10 @@ private static CqlTable buildCqlTable(Table table) throws StatusException {
     if (table.comment() != null) {
       cqlTableBuilder.putOptions("comment", table.comment());
     }
+    int ttl = table.ttl();
+    if (ttl != 0) { // any other markers (is -1 used as indicator?)
+      cqlTableBuilder.putOptions("ttl", String.valueOf(ttl));
```

Comment:
Not ideal to have to convert to `String`, but that's the mechanism we have. Unless we want to add an explicit `ttl` property for `CqlTable` or such.